models/schemas.yml

```diff
@@ -18,10 +19,11 @@ models:
   - name: fct_dbt_latest_full_model_executions
     description: A list of all models and executions times from the most recent, incremental run.
     columns:
-    - name: node_id
+    - name: latest_model_id
       description: Primary key.
       tests:
         - unique
+          severity:warn
```

Comment:
if this is actually unique now (🤞) we want it to be real error, not a warning so we know when its somehow not unique anymore